spring-cloud-task-core/src/test/java/org/springframework/cloud/task/listener/TaskLifecycleListenerTests.java

```diff
@@ -139,7 +139,7 @@ private void verifyTaskExecution(int numberOfParams, boolean update, Integer exi
 		assertEquals(exitCode, taskExecution.getExitCode());
 
 		if(exception != null) {
-			assertTrue(taskExecution.getExitMessage().length() > exception.getStackTrace().length);
+			assertTrue(taskExecution.getErrorMessage().length() > exception.getStackTrace().length);
```

Comment:
probably want to have the assertNull on the errorMessage on line 145.  its checking exit message.  will fix on merge.